spec/Pim/Bundle/DataGeneratorBundle/Generator/GeneratorSpec.php

```diff
@@ -15,9 +16,16 @@ function let(
         AttributeGenerator $attributeGenerator,
         FamilyGenerator $familyGenerator,
         ProductGenerator $productGenerator,
-        CategoryGenerator $categoryGenerator
+        CategoryGenerator $categoryGenerator,
+        AttributeOptionGenerator $attributeOptionGenerator
     ) {
-        $this->beConstructedWith($attributeGenerator, $familyGenerator, $productGenerator, $categoryGenerator);
```

Comment:
No need according to Benoit, it will be massively refactored so specs are useless here.